src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -57,7 +57,7 @@ public static unsafe void Ctor_CharPtr_OddAddressShouldStillWork()
                 // The address of a fixed byte[] should always be even
                 Debug.Assert(unchecked((int)pBytes) % 2 == 0);
                 char* pCh = (char*)(pBytes + 1);
-                
+
```

Comment:
Agh. I tried to avoid this whitespace stuff. I will revert the formatting which VS did automatically

Reply:
Reverted file - moved units to another file